proto/buf/alpha/registry/v1alpha1/convert.proto

```diff
@@ -29,7 +29,9 @@ enum ConvertFormat {
 service ConvertService {
   // Convert converts a serialized message according to
   // the provided type name using an image.
-  rpc Convert(ConvertRequest) returns (ConvertResponse);
+  rpc Convert(ConvertRequest) returns (ConvertResponse) {
+    option idempotency_level = NO_SIDE_EFFECTS;
```

Comment:
Maybe not this one?

(this is a drive-by, mainly marked all the ones that take an Image, does it make sense for this to support anything other than POST, maybe .. not sure).

Reply:
I think it just indicates they have no side effects, so they could be called as a GET. However, you might be right that we should leave off the bigger methods since there are limits to query parameters for most requests so even if we **could** send them as a GET, we probably shouldn't as we'd have to retry as a POST.

Reply:
FYI: we debated exposing an option to cap the size of GETs, and eventually decided not to export it in this release. It's still in the `connect-go` codebase and is fully tested, so we can easily export it if we need it.

Reply:
I think that would be very useful, but we might also want to tweak the guidance for when to use this feature to recommend it be used with methods that have smaller request messages that stay way below the limits.